### ISSUE TITLE
Incorrect operand access on x86 instruction vmovdqu

### ISSUE THREAD

**Issue by thsmdt** [capstone-engine/capstone#1821]
Hey there 😁

I came across the x86 instruction `vmovdqu` during a project of mine and discovered that both operands of this move are returned as read by capstone. Using the [x86 test program](https://github.com/capstone-engine/capstone/blob/next/tests/test_x86.c) I verified that with the output shown below:

```
****************
Platform: X86 64 (Intel syntax)
Code:0xc4 0xa1 0x7a 0x7f 0x44 0x27 0xf0 
Disasm:
0x1000:	vmovdqu	xmmword ptr [rdi + r12 - 0x10], xmm0
	Prefix:0x00 0x00 0x00 0x00 
	Opcode:0xc4 0xa1 0x7a 0x00 
	rex: 0x42
	addr_size: 8
	modrm: 0x44
	modrm_offset: 0x4
	disp: 0xfffffffffffffff0
	disp_offset: 0x6
	disp_size: 0x1
	sib: 0x27
		sib_base: rdi
		sib_index: r12
		sib_scale: 1
	op_count: 2
		operands[0].type: MEM
			operands[0].mem.base: REG = rdi
			operands[0].mem.index: REG = r12
			operands[0].mem.disp: 0xfffffffffffffff0
		operands[0].size: 16
		operands[0].access: READ
		operands[1].type: REG = xmm0
		operands[1].size: 16
		operands[1].access: READ
	Registers read: rdi r12 xmm0

```

This seems to be a bug, as according to [Intel® 64 and IA-32 Architectures Software Developer’s Manual - Vol. 2B 4-67](https://www.intel.com/content/dam/www/public/us/en/documents/manuals/64-ia-32-architectures-software-developer-instruction-set-reference-manual-325383.pdf#page=719) the first operand is written to and should thus be returned as such.

I hope this is helpful 😊

**Comment by aquynh** [comment]
make sure that you use the "next" branch code.

**Comment by thsmdt** [comment]
Thanks for following up, @aquynh 🙂 - I appreciate it.

Up until now, I used a precompiled v4.0.2; however, I just compiled the source from the current `next` branch (rev. c7c665f8f38276dbbf9bd417c89bc17cefe08cdb) and could still reproduce the above issue.

**Comment by kabeor** [comment]
Fixed in #1823 .
